_posts/2023-03-13-geoserver-2-23-RC1-released.md

```diff
@@ -0,0 +1,92 @@
+---
+author: Gabriel Roldan
+date: 2023-03-13
+layout: post
+title: GeoServer 2.23-RC1 Release
+categories:
+- Announcements
+tags:
+- Release
+release: release_223
+version: 2.23-RC1
+jira_version: 16862
```

Comment:
You will need to uidsye the config yml to describe the next version and change what is master and stable

Reply:
done, check the pr description